docs/feature_flags.md

```diff
@@ -194,3 +194,15 @@ won't work when you push OTLP metrics.
 
 Enables PromQL functions that are considered experimental and whose name or
 semantics could change.
+
+## Created Timestamps Zero Injection
+
+`--enable-feature=created-timestamp-zero-ingestion`
+
+Enables ingestion of Created Timestamps as injection of samples with 0 as value. 
+
+Created timestamps are only supported by the traditional Prometheus Protobuf protocol. When enabling this feature,
+therefore, the default scrape protocol will be switched to Prometheus Protobuf.
```

Comment:
```suggestion
```suggestion
Currently Prometheus supports created timestamp only on the traditional Prometheus Protobuf protocol (WIP for other protocols). As a result, when enabling this feature, the Prometheus protobuf scrape protocol will be prioritized (See `scrape_config.scrape_protocols` settings for more details).
```